crates/aranya-client/src/error.rs

```diff
@@ -1,25 +1,146 @@
-/// Errors that could occur in the Aranya client.
+use aranya_daemon_api::AfcId;
+use aranya_fast_channels::Version;
```

Comment:
This would be more clear:
```suggestion
use aranya_fast_channels::AfcVersion;
```

Reply:
Version is only use'd because otherwise it would make VersionMismatch take up 4 lines formatted